File: app/src/main/java/com/intetm/comicbreeze/service/database/model/Page.java
package com.intetm.comicbreeze.service.database.model;


import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity
public class Page {

    @PrimaryKey
    public Long id;

    @ColumnInfo(name = "comic")
    public Integer comic;

    @ColumnInfo(name = "index")
    public int index;

    @ColumnInfo(name = "url")
    public String url;

}

File: app/src/main/java/com/intetm/comicbreeze/service/database/model/Page.kt
package com.intetm.comicbreeze.service.database.model


import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity
class Page {

    @PrimaryKey
    var id: Long? = null

    @ColumnInfo(name = "comic")
    var comic: Int? = null

    @ColumnInfo(name = "index")
    var index: Int = 0

    @ColumnInfo(name = "url")
    var url: String? = null

}
